simulacrum/__init__.py

```diff
@@ -1,9 +1,2 @@
-import pandas as pd
-from faker import Faker
-import logging
-import numpy as np
-from datetime import datetime
-
-from .dataset import DataSet
-from .simdata import create
+from .dataset import *
```

Comment:
It could be better to avoid star import everywhere.

I prefer to see `from .dataset import create`
If the user want another function, he can use the complete path to dataset.